packages/application-shell/src/components/quick-access/quick-access.spec.js

```diff
@@ -1166,7 +1165,7 @@ describe('QuickAccess', () => {
 
       const searchInput = getByTestId('quick-access-search-input');
       fireEvent.change(searchInput, { target: { value: searchTerm } });
-      await waitForElement(() => getByText('Open Dashboard'));
+      await waitForElement(() => getByText('Open Manage Organizations'));
```

Comment:
The dashboard is bound to orders now, just as in the MC. As a result we can't wait for it when we intend to test if orders are hidden.